src/components/RocketProfile/RocketProfile.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './RocketProfile.css';
+
+function RocketProfile() {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const bookedRockets = rockets.filter((rocket) => rocket.reserved);
+
+  return (
+    <div>
+      <h2>My Rockets</h2>
+      <table className="rocket-profile-table">
+        <tbody>
+          {bookedRockets.length === 0 ? (
+            <tr>
+              <td className="no-rocket-placeholder">No Rockets Added</td>
+            </tr>
+          ) : (
+            bookedRockets.map((rocket) => (
+              <tr key={rocket.id}>
+                <td>{rocket.name}</td>
+              </tr>
+            ))
```

Comment:
- [OPTIONAL ] I can see that your tests are passing :+1: , but I noticed that you are encountering a `!warning` related to `prop` types in your code. and noticed an issue related to the` prop` types in your RocketInfo component. The error message indicates that the `id` prop is expected to be of type `string`, but it's being supplied as a number. check in the screenshot below :point_down: 
![Screenshot from 2023-09-09 14-16-47](https://github.com/pushkar1995/space-travelers-hub/assets/88894525/752748f3-8593-4aaf-b52e-32237c3b2ba6)


**If you find that the id prop is defined as a number in the propTypes, update it to expect a string. :wink:**

```RocketInfo.propTypes = {
  // Other prop types...
  id: PropTypes.string.isRequired,
  // then you can add Other prop types...
};
```